src/pgrepr/src/oid.rs

```diff
@@ -63,9 +63,10 @@ pub const FUNC_MZ_SLEEP_OID: u32 = 16_434;
 pub const FUNC_MZ_SESSION_ID_OID: u32 = 16_435;
 pub const FUNC_MZ_UPTIME_OID: u32 = 16_436;
 pub const FUNC_MZ_WORKERS_OID: u32 = 16_437;
+// Deprecated
 pub const TYPE_APD_OID: u32 = 16_438;
```

Comment:
That's a good question--wasn't sure how "pure" we wanted to be about OID reassignment. If you're comfortable with it getting axed, sgtm

Reply:
Ah yeah, sorry, probably good to keep the OID reserved but can we change the symbol to `__DEPRECATED_TYPE_APD_OID` so that folks don't use it without thinking twice?